manager/ocpi/server.go

```diff
@@ -371,3 +429,15 @@ func (s *Server) GetTokensPageFromDataOwner(w http.ResponseWriter, r *http.Reque
 func (s *Server) PostRealTimeTokenAuthorization(w http.ResponseWriter, r *http.Request, tokenUID string, params PostRealTimeTokenAuthorizationParams) {
 	w.WriteHeader(http.StatusNotImplemented)
 }
+
+func extractChargeStationId(evseId string) (string, error) {
+	pattern := `^[a-zA-Z]{2}\*[a-zA-Z0-9]{3}\*([a-zA-Z0-9]+)(-[a-zA-Z0-9]+)?$`
```

Comment:
The * separators are all optional. We should strip those out before storing any EVSE ID. An EVSE Id MUST have the character 'E' as the first (non-optional separator) character following the party id (which will not be part of the CS-ID) and I don't believe hyphen is a valid character.

Reply:
Ah, thank you, I've misread the spec, got it now, I think. I'll change the pattern to `^[a-zA-Z]{5}E([a-zA-Z0-9]+)$` then.